assets/aurora.tokenlist.json

```diff
@@ -55,6 +55,28 @@
         "ethereum"
       ]
     },
+    {
+      "chainId": 1313161554,
+      "address": "0xdc7acde9ff18b4d189010a21a44ce51ec874ea7c",
+      "symbol": "agEUR",
+      "name": "agEUR",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/ageur.svg",
+      "tags": [
+        "ethereum"
+      ]
+    },
+    {
+      "chainId": 1313161554,
+      "address": "0xb7e3617adb58dc34068522bd20cfe1660780b750",
+      "symbol": "ANGLE",
+      "name": "ANGLE",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/angle.svg",
+      "tags": [
+        "ethereum"
+      ]
+    },
```

Comment:
Please don't modify that file, it's regenerating with the tool when the PR is merged

Reply:
ok, got it. sorry about that :)